src/main/java/com/lmax/disruptor/RingBuffer.java

```diff
@@ -24,14 +27,39 @@
  *
  * @param <E> implementation storing the data for sharing during exchange or parallel coordination of an event.
  */
-public final class RingBuffer<E> implements Cursored, DataProvider<E>
+abstract class RingBufferPad
 {
-    public static final long INITIAL_CURSOR_VALUE = Sequence.INITIAL_VALUE;
+    protected long p1, p2, p3, p4, p5, p6, p7;
```

Comment:
The RingBuffer fields:

    private final long indexMask;
    private final Object[] entries;
    protected final int bufferSize;
    protected final Sequencer sequencer;

Could end up sharing a cache line with some other data.